plugins/signalmonitor/signalhistorydelegate.cpp

```diff
@@ -47,10 +47,10 @@ SignalHistoryDelegate::SignalHistoryDelegate(QObject *parent)
     , m_visibleOffset(0)
     , m_visibleInterval(15000)
     , m_totalInterval(0)
+    , m_fps(0)
 {
     connect(m_updateTimer, SIGNAL(timeout()), this, SLOT(onUpdateTimeout()));
-    m_updateTimer->start(1000 / 25);
-    onUpdateTimeout();
+    setFPS(15);
```

Comment:
That would indeed help more also.
Still if you look closer in the code you will see that now no network traffic should be emitted for the events by default (You have to tick an object row so it become monitored and start transmitting signals data). The amont of data sent by default should not be more than the object one itself.
What is still an issue is the high frequency clock time of the signal monitor interface this is it struggling the network traffic.